File: TYContrastTest/TYContrastTest/ViewController.m
//
//  ViewController.m
//  TYContrastTest
//
//  Created by Mac on 16/3/18.
//  Copyright © 2016年 TY. All rights reserved.
//

#import "ViewController.h"

typedef NS_ENUM(NSInteger, TYTestEnum) {
    TYTestEnum1 = 0,
    TYTesta = 1
};

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    NSArray *nameArray = @[@"fda", @"fdsa", @"jkl"];
    for (NSString *name in nameArray) {
        NSLog(@"%@", name);
    } UIButton
}

- (instancetype)init {
    if (self = [super init]) {
        self.view.backgroundColor = [UIColor redColor];
    }
    return self;
}

- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil {
    if (self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil]) {
        
    }
    return self;
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)test {
    NSInteger fda = 1;
    switch (fda) {
        case 0: {
        
        }
            
            break;
        case 1: {
        
        }
            break;
        default:
        {
        
        }
            break;
    }
}

@end

File: TYSwift/TYSwift/ViewController.swift
//
//  ViewController.swift
//  TYSwift
//
//  Created by Mac on 16/3/18.
//  Copyright © 2016年 TY. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    
    func test1() -> Int? {
        let testArray = [44, 55, 32, 53]
        
        var teamScore = 0
        for score in testArray {
            if score > 50 {
                teamScore += 3
            } else {
                teamScore -= 1
            }
        }
        
        return 1;
    }
    
}

